redfish_utilities/arguments.py

```diff
@@ -0,0 +1,43 @@
+import argparse
```

Comment:
We'll need the DMTF copyright above this line (just need to copy/paste from another file in here).